test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Waiters.cs

```diff
@@ -44,6 +44,8 @@ public static void WaitForFileToBeReadable(string file, TimeSpan timeout)
 
         public static void WaitForProcessToStop(int processId, TimeSpan timeout, bool expectedToStop, string errorMessage)
         {
+            Console.WriteLine($"Waiting for process {processId} to stop...");
```

Comment:
Remove `WriteLine()`s before checking in. These don't help when the tests are working and just increase the noise level.